docs/utils/sessions.md

```diff
@@ -309,15 +318,15 @@ const { getSession, commitSession, destroySession } =
 export { getSession, commitSession, destroySession };
 ```
 
-## `createFileSessionStorage` (node)
+## `createFileSessionStorage` (Node.js, Deno)
```

Comment:
Let's also update this in the list at the top of this page

Reply:
Done!